### DISCUSSION TITLE
.NET MAUI Blazor Hybrid multi window implementation

### DISCUSSION THREAD

**Discussion by kawishbit** [dotnet/maui#24801] [Q&A]
Hi everyone, 

I want to implement multi window feature on my simple windows app. Basically what I want to do is that I want to put a cog icon on the title bar, and when this icon is clicked it will open the /config page on a new window while also maintaining the state of the app. 

The best example I can give for it is this app: https://github.com/yaakapp/app . As you can see, this app has a cog icon on the top right corner, and when you click this icon, it will open a new window for the config page.

Is this possible using blazor hybrid? Thanks for your help

**Comment by roke-julian-lockhart** [comment]
@kawishbit, is [`stackoverflow.com/a/78139472`](https://stackoverflow.com/a/78139472/9731176) of use? If not, [`issues/16317#issuecomment-1649259715`](https://github.com/dotnet/maui/issues/16317#issuecomment-1649259715) might be.
